pepsico/projections/layout.py

```diff
@@ -283,22 +283,10 @@ def map_layout():
 
 
 def results_layout():
-    return dbc.Tabs(
-        [
-            dbc.Tab(
-                [
-                    dbc.Row(
-                        [
-                            dbc.Col(
-                                dbc.Spinner(
-                                    dcc.Graph(id="local_graph"),
-                                )
-                            ),
-                        ]
-                    )
-                ],
-                label="Local History and Projections",
-            )
-        ],
-        className="mt-4",
-    )
+    return dbc.Tabs([dbc.Tab([
```

Comment:
I don't understand why we're using Tabs if there's only one tab. Do you plan to add more?

Reply:
Some other Maprooms have more than one. This is just repeating the Structure. I could remove it but I'd still want its label to indicate the "Local" section without jumping directly to the figure title, and without clogging further the figure title with information should there be no intermediary. So might as well keep the lonely Tab?